docs/dg/dev/backend-development/session-handlers.md

```diff
@@ -18,7 +18,7 @@ The following session handlers are currently available:
 
 * File
 * Couchbase
-* MySql
+* MySql (**OUTDATED!**)
```

Comment:
Why not just remove it?

Reply:
Because we deliver it and it is part of `Session` module.